sdk/jobs/automl-standalone-jobs/automl-image-classification-multiclass-task-fridge-items/automl-image-classification-multiclass-task-fridge-items-mlflow.ipynb

```diff
@@ -1106,6 +1124,99 @@
     "plt.show()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Generate the Scores and Explanations\n",
+    "- Explainability methods:\n",
```

Comment:
I'd replace this with a reference to the documentation (the one to be done) and something like: 

"AutoML deployed models now support explainablity. Please refer to the documentation [link here] for more details."